node/components/bft/src/chonky_bft/proposer.rs

```diff
@@ -27,15 +26,15 @@ pub(crate) async fn run_proposer(
 
         // Create a proposal for the given justification, within the timeout.
         let proposal = match create_proposal(
-            &ctx.with_timeout(VIEW_TIMEOUT_DURATION),
+            &ctx.with_timeout(cfg.view_timeout),
             cfg.clone(),
             justification,
         )
         .await
         {
             Ok(proposal) => proposal,
             Err(ctx::Error::Canceled(_)) => {
-                tracing::error!("run_proposer(): timed out while creating a proposal");
+                tracing::warn!("run_proposer(): timed out while creating a proposal");
```

Comment:
maybe just info? Afaiu it is expected to have infrequent blocks on some chains.

Reply:
True, but maybe in that case the chain should be warned to increase the timeout. Overall, I don't think having a lot of timeouts should be considered normal operation.